crates/generated_parser/src/ast_builder.rs

```diff
@@ -4979,4 +5189,35 @@ impl<'alloc> AstBuilder<'alloc> {
 
         false
     }
+
+    fn check_labelled_statement(
+        &mut self,
+        label: &arena::Box<'alloc, Label>,
+        body: &Statement<'alloc>,
+    ) -> Result<'alloc, ()> {
+        let start_offset = label.loc.start;
+        let end_offset = label.loc.end;
```

Comment:
`end_offset` name here sounds like the end of the entire labelled statement.
can it be `label_end_offset` or something that clarifies it's not the end of body ?